src/schema/rules/sidecars/emg.yaml

```diff
@@ -72,6 +72,8 @@ EMGRecommended:
     EMGPlacementSchemeDescription: recommended
     EpochLength: recommended
     HardwareFilters: recommended
+    Preamplification: recommended
+    Gain: recommended
```

Comment:
Note that `recommended` means WARN on absent. Is this a thing that indicates a likely problem if missing?